keras_nlp/models/bart/bart_backbone.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""BART backbone model."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers.position_embedding import PositionEmbedding
+from keras_nlp.layers.transformer_decoder import TransformerDecoder
+from keras_nlp.layers.transformer_encoder import TransformerEncoder
+from keras_nlp.models.backbone import Backbone
+
+
+def bart_kernel_initializer(stddev=0.02):
+    return keras.initializers.TruncatedNormal(stddev=stddev)
+
+
+@keras.utils.register_keras_serializable(package="keras_nlp")
+class BartBackbone(Backbone):
```

Comment:
I wish they had chosen a name that wasn't so visually similar to Bert. This is going to confuse me so much :P

Reply:
:joy:

Reply:
I wish they explained what BART stands for....

![image](https://user-images.githubusercontent.com/1141045/213083438-15364174-4fa7-4a22-8353-e10254d8ea70.png)

Reply:
BERT -> Bidirectional Encoder Representations from Transformers
BART -> Bidirectional **Auto-encoder** Representations from Transformers?

Guessing...:P

Reply:
Interesting! Couldn't find it in the paper